### PR TITLE
[REF] Change the linked examples to recipe repo

### LINKED ISSUE #144: Switch local node setup instructions to use `recipes` repo
### Description

Local node setup instructions should be updated to use config files from the newly created https://github.com/neurobagel/recipes instead of repo-specific files.

### Relevant documentation pages

_No response_

### Additional details

_No response_

TODO:
- [x] https://neurobagel.org/infrastructure/#clone-the-api-repo should instead point to recipes

### PR DESCRIPTION
**Pull request by surchs**
- Closes #144 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Checks pass

### CODE REVIEW

File: docs/infrastructure.md
Comment (alyssadai) [inline docs/infrastructure.md:1]: It looks like there are a few sections in this page where files from the `neurobagel/api` repo is still referred to for setup, and need to be updated to use corresponding files from `neurobagel/recipes` instead:
- "Uploading data to the graph"
- "Adding vocabulary files to the graph database"
  - Related to an open issue https://github.com/neurobagel/recipes/issues/14
- "Updating your Neurobagel API configuration"
  > For example, the [.template-env](https://github.com/neurobagel/api/blob/main/.template-env) file in the Neurobagel API repo...

In the "Docker Compose" section, I would also update
> Run the following in the repository root (where the `docker-compose.yml` file is)

to

> Run the following in the directory containing both the `docker-compose.yml` file from the [`local_nodes` recipe](link) and the `.env` file you just created.

**Review by alyssadai — CHANGES_REQUESTED** [review]
Hey @surchs, sorry for the delay with this review!

The changes are looking good, but I noticed there are still a few remaining sections on this docs page that refer to files from the neurobagel/api repo (I did a quick search for "api repo" to double check), which should also be updated as part of this PR.

Since I can't seem to comment directly on lines that weren't changed, I've summarized these in my last comment below!

**Comment by surchs** [comment]
See also:
- https://github.com/neurobagel/api/pull/272
- https://github.com/neurobagel/api/pull/272

File: docs/infrastructure.md [hunk 1]
```diff
@@ -641,7 +650,7 @@ To make the Neurobagel API accessible by a frontend tool such as our [browser qu
 you must explicitly specify the origin(s) for the frontend using `NB_API_ALLOWED_ORIGINS` in `.env`.
 For detailed instructions regarding the query tool see [Running cohort queries](query_tool.md).
```
Comment (alyssadai) [inline docs/infrastructure.md:651]:
```suggestion
(Detailed instructions for using the query tool can be found in [Running cohort queries](query_tool.md).)
```

File: docs/infrastructure.md [hunk 2]
```diff
@@ -641,7 +650,7 @@ To make the Neurobagel API accessible by a frontend tool such as our [browser qu
...
 
-For example, the [`.template-env`](https://github.com/neurobagel/api/blob/main/.template-env) file in the Neurobagel API repo assumes you want to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).
+For example, the [`template.env`](https://github.com/neurobagel/recipes/blob/main/local_node/template.env) file in the Neurobagel API repo assumes you want to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).
```
Comment (alyssadai) [inline docs/infrastructure.md:653]:
````suggestion
For example, add the following line to your `.env` file to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).

```bash
NB_API_ALLOWED_ORIGINS="http://localhost:3000 http://127.0.0.1:3000"
```
````

File: docs/infrastructure.md [hunk 3]
```diff
@@ -522,11 +531,11 @@ You can choose to omit the flag or explicitly specify `--no-clear-data` (default
 
 The participant variables modeled by Neurobagel are named using Neurobagel's own vocabulary (for more information, see this page on [controlled terms](./term_naming_standards.md)).
 This vocabulary, which defines internal relationships between vocabulary terms, 
-is serialized in the file [`nb_vocab.ttl`](https://github.com/neurobagel/api/blob/main/vocab/nb_vocab.ttl) available from the `neurobagel/api` repository.
+is serialized in the file [`nb_vocab.ttl`](https://github.com/neurobagel/recipes/blob/main/vocab/nb_vocab.ttl) available from the `neurobagel/recipes` repository.
 If you have cloned the api repository, you will already have downloaded the vocabulary file.
```
Comment (alyssadai) [inline docs/infrastructure.md:535]:
```suggestion
If you have cloned this repository, you will already have downloaded the vocabulary file.
```

**Review by alyssadai — APPROVED** [review]
Thanks for the changes @surchs! Some final corrections below with regards to repo naming, and I think we're gtg 🧑‍🍳

File: docs/infrastructure.md [hunk 4]
```diff
@@ -522,11 +531,11 @@ You can choose to omit the flag or explicitly specify `--no-clear-data` (default
...
 
 **The `nb_vocab.ttl` file should be added to every created Neurobagel graph database.**
-This can be done using the same script we used to upload the dataset JSONLD files, [`add_data_to_graph.sh`](https://github.com/neurobagel/api/blob/main/add_data_to_graph.sh), which adds all `.ttl` and/or `.jsonld` files in a given directory to the specified graph.
+This can be done using the same script we used to upload the dataset JSONLD files, [`add_data_to_graph.sh`](https://github.com/neurobagel/recipes/blob/main/scripts/add_data_to_graph.sh), which adds all `.ttl` and/or `.jsonld` files in a given directory to the specified graph.
 
 Run the following code (assumes you are in the `api` directory):
```
[the remark this answers is not in the document]
Comment (surchs) [reply docs/infrastructure.md]: Thanks for finding these! I thought I had caught them all.

**Comment by surchs** [comment]
The dead links check failing is expected, because these links are anticipating that https://github.com/neurobagel/recipes/pull/15 will be merged as is.

**Pull request merged**